docs/csharp/language-reference/operators/null-parameter-check.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "!! (null validation) operator - C# reference"
+description: "Learn about simplified null-parameter checks. This operator instructs the compiler to add runtime checks that the argument used for a parameter isn't null."
+ms.date: 03/30/2022
+f1_keywords: 
+  - "!!_CSharpKeyword"
```

Comment:
@RikkiGibson Now Roslyn can support `!!` in F1 :)

Reply:
Oh, great. Do we need to wire anything up to make that happen?

Reply:
@RikkiGibson Likely yes we'll need to update [CSharpHelpContextService](https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs). [Tests](https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/CSharp/Test/F1Help/F1HelpTests.cs)